datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -157,9 +157,8 @@ export const HomePageHeader = () => {
     }, [suggestionsData]);
 
     const onSearch = (query: string, type?: EntityType, filters?: FacetFilterInput[]) => {
-        if (!query || query.trim().length === 0) {
-            return;
-        }
+        if (!query.trim() && !selectedQuickFilter) return;
```

Comment:
Crazy idea: What if anytime you hit enter we take you to explore all? So basically remove these checks altogether?

Reply:
I like it.

Reply:
One unrelated bug I noticed is when I hit enter with 1 or 2 characters it always finds no results.

Reply:
It's probably gonna take me a bit to make that change, so I would rather decouple that fix from this PR and move forward with this one as-is.

Reply:
^ I would ignore the unrelated bug I believe that is just how our search works where you need at least 3 characters because we break our search result terms into 3 character chunks